llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -928,8 +928,19 @@ void VPlan::prepareToExecute(Value *TripCountV, Value *VectorTripCountV,
 
   IRBuilder<> Builder(State.CFG.PrevBB->getTerminator());
   // FIXME: Model VF * UF computation completely in VPlan.
```

Comment:
Would it be possible to handle this FIXME now that VF is assigned a VPValue?

Reply:
Yes, would either require introducing a `UF` placeholder or introducing during explicit interleaving, either way probably better as follow-up?

Reply:
Yep